packages/karma-puppeteer-launcher/src/index.cjs

```diff
@@ -35,7 +35,8 @@ function puppeteerBrowser(baseBrowserDecorator, config) {
     const defaultPuppeteerOptions = {
       product: 'chrome',
       slowMo: 0,
-      dumpio: true,
+      // Only enable for debugging
+      // dumpio: true,
```

Comment:
How do we enable that for debugging? Is it something we use when running tests locally or is it when debugging the test setup itself? If used regularly, should we make it a CLI flag?

Reply:
If you do `dumpio: true` then you will see more logs (including console.logs) it is used rarely on CI and either way you need to commit this change to CI and wait for results (and then remove it) so with the CLI flag it would be the same.